extensions/amp-pan-zoom/0.1/amp-pan-zoom.js

```diff
@@ -181,9 +181,11 @@ export class AmpPanZoom extends AMP.BaseElement {
 
   /** @override */
   layoutCallback() {
-    this.content_.classList.add('i-amphtml-pan-zoom-child');
-    return this.resetContentDimensions_()
-        .then(() => this.setupGestures_());
+    return this.resetContentDimensions_().then(() => {
+      if (!this.gestures_) {
```

Comment:
nit: [prefer positive if conditions ](http://go/tott-530)(internal link) and shortcut `return`s